test/test_force.js

```diff
@@ -267,7 +265,18 @@ function createDeploySfdxPluginPackage(tmpDir) {
     utils.runProcessThrowError('npm install --prefix ' + tmpDir + ' ' + 'sfdx-mobilesdk-plugin-' + SDK.version + '.tgz');
     utils.runProcessCatchError('sfdx plugins:uninstall sfdx-mobilesdk-plugin');
     utils.logInfo('Sfdx linking sfdx-mobilesdk-plugin', COLOR.green);
+
```

Comment:
This is the only non-white space modification to the test file and really is just to support the oclif manifest.